flink-runtime-web/web-dashboard/src/app/share/pipes/humanize-watermark.pipe.ts

```diff
@@ -24,7 +24,9 @@ import { LONG_MIN_VALUE } from 'config';
 })
 export class HumanizeWatermarkPipe implements PipeTransform {
   transform(value: any): any {
-    if (isNaN(value) || value <= LONG_MIN_VALUE) {
+    if (isNaN(value)) {
+      return 'Watermarks are only available if EventTime is used';
+    } else if (value <= LONG_MIN_VALUE) {
       return 'No Watermark';
```

Comment:
I am not sure whether this change does the trick because if a user selects processing time as the time characteristic, then Flink will report `Long.MIN_VALUE` back.

Reply:
Thanks for the info, Till.  I add the condition when value<= LONG_MIN to display new message.